src/components/Link.js

```diff
@@ -1,9 +1,20 @@
 import React from "react"
-import { Link as GatsbyLink } from "gatsby"
+import { Link as GatsbyLink, withPrefix } from "gatsby"
 import { Link as ScrollLink } from "react-scroll"
+import { useLocation } from "@reach/router"
 import PropTypes from "prop-types"
 
 const Link = ({ url, children, ...props }) => {
+  /* eslint-disable no-var */
+  try {
+    var location = useLocation()
+  } catch (error) {
+    var location = window.location
+  }
```

Comment:
As ugly as the pattern I showed was, I think `let` is actually still better than using `var` here. The scoping characteristics of `var` are a little weird, and `let` is very explicit about what we're doing.

In a perfect world I'd want this to be a `const`, but that won't work with the gymnastics we have to pull.

Reply:
I agree about using the `let` pattern actually! I just went with this because using the `let` pattern in the tbtc-website repo caused a compilation error that broke the build. However, I tried it with `let` here and what do you know, it's completely fine here.  Maybe because in the tbtc-repo it was in a component wrapped in a `StaticQuery`? Or the gatsby version is different? Not going to investigate right now, but I updated this!

Reply:
Weird. Another fun surprise for another day lolol.